.github/workflows/int_test.yml

```diff
@@ -36,27 +36,32 @@ on:
         required: false
         default: false
 
-env:
-  CLUSTER_NAME: "az-iot-ops-test-cluster-${{ github.run_id }}-${{ github.run_attempt }}"
-  KV_NAME: "opstestkv-${{ github.run_id }}-${{ github.run_attempt }}"
-  CLUSTER_RG: ${{ inputs.resource_group }}
-  CUSTOM_LOCATIONS_OID: "51dfe1e8-70c6-4de5-a08e-e18aff23d815"
-  EXTENSION_SOURCE_DIRECTORY: "./azure-iot-ops-cli-extension"
-  K3S_VERSION: "v1.28.5+k3s1"
-
 permissions:
   # required for OpenID federation
   contents: 'read'
   id-token: 'write'
 
 jobs:
   integration_test:
+    env:
+      CLUSTER_NAME: "az-iot-ops-test-cluster${{ github.run_id }}${{ matrix.feature }}"
+      # TODO - KV Name has a hard limit of 24 chars
+      KV_NAME: "opstestkv${{ github.run_id }}${{ matrix.feature }}"
```

Comment:
to save on characters, we can do at least one of the following:
- `opstestkv` -> `opstkv` (saves 3)
- truncate the run id ( variable saves) -> there is a `truncate()` command
- share the kv between the matrix features (saves up to 3?)
- use tags to specify what tests were run

Reply:
Good ideas - here's what's coming in the updated commit:

- new prefix will just be `opskv`
- using run # instead of run ID - it's only for runs of the specific workflow so it starts at a 2-3 digit number instead of 10
- kv is created at the start and shared between matrix features